api/src/resolvers/index.js

```diff
@@ -1,9 +1,23 @@
 const { signIn, searchUsers } = require("./user");
-const { queryEventByID, searchEvents } = require("./event");
-const { querySeminarByID, searchSeminars } = require("./seminar");
+const { queryEventByID } = require("./event");
+const { querySeminarByID } = require("./seminar");
 const { queryAnnouncementByTypeID } = require("./announcement");
+const { searchEventsAndSeminars, getTotalCount } = require("./searchResults");
 
 const rootResolvers = {
+  SearchResult: {
+    __resolveType(obj) {
+      if (obj.creator_id) {
+        return "Event";
+      }
+
+      if (obj.event_id) {
```

Comment:
This resolver will only ever be searching through events and seminars, do we need to check both or can we just do else?